docs/src/index.md

```diff
@@ -337,8 +337,8 @@ example the result of `@transform(df, @byrow :y = f(:x))` and
 
 ## Propagating missing values with `@passmissing`
 
-Many Julia functions to not automatically propagate missing values. For instance, 
-`parse(Int, missing)` will error. 
+Many Julia functions do not automatically propagate missing values. For instance, 
+`parse(Int, missing)` will throw an error. 
```

Comment:
I think "will error" is fine. 
```suggestion
`parse(Int, missing)` will error. 
```

Reply:
I can leave it as is and close the PR, but I found the phrase distracting.

English is my second language, but I believe "error" is a noun ([MW on "error"](https://www.merriam-webster.com/dictionary/error#synonyms)) and the verb would be "to err" ([MW on "err"](https://www.merriam-webster.com/dictionary/err). 

I've also searched Google for the phrases "to error" and "will error" and haven't found any reputable reference.

Reply:
Updated as per @pdeffebach suggestion.